docs/en/observability/synthetics-troubleshooting.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[synthetics-troubleshooting]]
+= Troubleshooting Synthetics
+
+++++
+<titleabbrev>Troubleshooting</titleabbrev>
+++++
+
+[discrete]
+[[synthetics-troubleshooting-local-debugging]]
+= Local debugging
+
+Use the `DEBUG=synthetics` environment variable to capture Synthetics agent logs when
+running Synthetic tests locally with the <<synthetics-command-reference,Synthetics CLI>>.
+
+[discrete]
+[[synthetics-troubleshooting-common-issues]]
+= Common issues
+
+[discrete]
+[[synthetics-troubleshooting-missing-api-key]]
+== Missing API key
+
+Missing / incorrect API key (should get notification, but how to regenerate if not getting results)
```

Comment:
I recently created https://support.elastic.co/knowledge/e443de71 for this - does this give you enough info, or do you need something else?

Reply:
Thanks! This is very helpful. I took the information there and edited it down to:

1. What the user is seeing
2. A _brief_ explanation of why they're seeing it
3. How to fix it

I also updated the heading from "Missing API key" --> "Monitors stopped running after upgrading to 8.8.0 or above" because I don't think it will be obvious for users experiencing this issue to look for mentions of an API key since that's happening behind-the-scenes.

I can also add more details from the knowledge base page on why this happens and put it in a note or collapsible block as an aside (for those users who want more information) while maintaining a quick path to the fix. Let me know what you think!

Lastly, can you help me find the exact language used in the error message users will see if they don't have admin permissions and try to visit the Synthetics app before an admin does? That way if someone Googles the exact error message, this should come up.

Reply:
@colleenmcginnis this is what a user will see if the Synthetics app has never been visited by an admin, and can't be enabled:

![image](https://github.com/elastic/observability-docs/assets/70908889/e81fae63-6d9d-4137-9db4-6789aec5b0aa)

(The _Read the docs_ link points to https://www.elastic.co/guide/en/observability/current/synthetics-get-started-ui.html#uptime-set-up-prereq)